recipes/simbody/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+source activate "${CONDA_DEFAULT_ENV}"
+
+mkdir build
+cd build
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+	VIZ=on
+	SHARED_EXT=so
+	# TODO: This test is failing for a yet-to-be-determined reason. See
+	# https://github.com/simbody/simbody/issues/400 for more details. Once
+	# that is figured out then this test should be enabled.
+	SKIP_TEST="-E TestCustomConstraints"
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+	VIZ=off
```

Comment:
Also, it seems that simbody includes glut for windows: https://github.com/simbody/simbody/tree/Simbody-3.5.3/Simbody/Visualizer/simbody-visualizer/glut32